src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  display: grid;
+  grid-template-columns: #{$card-width};
+  gap: 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, #{$card-width});
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, #{$card-width});
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, #{$card-width});
+  }
```

Comment:
great job, you can also try to use mixin there like this if you want  @mixin col-width($width, $col) {
  @media (min-width: $width) {
    grid-template-columns: repeat($col, $card-width);
  }
}

Reply:
Thanks! Now I better understand how mixins work :)